pkg/healthz/healthz_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package healthz_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"testing"
+)
+
+func TestHealthz(t *testing.T) {
```

Comment:
It's not clear to me why these explicit test wrappers are needed. What do we gain?